src/NServiceBus.WindowsPerformanceCounters/PerformanceCounters.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using Settings;
+
+    public class PerformanceCounters
```

Comment:
We should add API docs and configure the project to fail on compile if public types don't have comments

https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/NServiceBus.Core.csproj#L30-L32